<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MesonProjectManager::Internal::ToolItemSettings</class>
 <widget class="QWidget" name="MesonProjectManager::Internal::ToolItemSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>70</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="_nameLbl">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="_pathLbl">
     <property name="text">
      <string>Path:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Utils::PathChooser" name="mesonPathChooser" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="mesonNameLineEdit"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
